clusterloader2/testing/load/deployment.yaml

```diff
@@ -28,18 +36,22 @@ spec:
         svc: {{.SvcName}}-{{.Index}}
 {{if and .EnableDNSTests $USE_ADVANCED_DNSTEST}}
         dns-test: dnsperfgo
+{{else}}
+  {{if $RunNetPolicyTest}}
+        {{$TargetLabelKey}}: {{$TargetLabelValue}}
+  {{end}}
 {{end}}
     spec:
       hostNetwork: {{$HostNetworkMode}}
       containers:
 {{if .EnableDNSTests}}
-{{if $USE_ADVANCED_DNSTEST}}
+  {{if $USE_ADVANCED_DNSTEST}}
       - image: gcr.io/k8s-staging-perf-tests/dnsperfgo:v1.3.0
         ports:
         - containerPort: 9153
           name: dnsperfmetrics
           protocol: TCP
-{{else}}
+  {{else}}
       - image: gcr.io/k8s-staging-perf-tests/dnsperfgo:v1.2.0
 {{end}}
```

Comment:
If we change the indentation, please indent this line as well for consistency's sake.